pkg/controller/s2irun/ksbuilder_jobs.go

```diff
@@ -22,6 +23,55 @@ const (
 	NodeAffinityValue = "ci"
 )
 
+func (r *ReconcileS2iRun) NewRegularClusterRole() *v1.ClusterRole {
+	cr := &v1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: RegularClusterRoleName,
+		},
+		Rules: []v1.PolicyRule{
```

Comment:
update pod not job. pod should not exceed its power to control job. it will increase the burden of k8s controller-manager and also our operator.